### ISSUE TITLE
Problem: Can't edit my own record in custom made module related to Company

### ISSUE THREAD

**Issue by serbiaserbia** [YetiForceCompany/YetiForceCRM#3354]
Yesterday everything worked as it should. Today I can not new edit record.

I made a new module (Survey) and connect it with Company module.
I made workflow - when I create Company record then created and a new record in the module Survey. It is important ability to edit new Survey record - to fill the survey.

When try to edit, message is:
![prob4](https://cloud.githubusercontent.com/assets/19521910/17543473/35599b22-5ed1-11e6-846a-169c7249f79f.png)
![prob1](https://cloud.githubusercontent.com/assets/19521910/17543445/fc7d10a4-5ed0-11e6-8ad5-1a688b1dc9d7.png)
When create new record from Survey module I have no ability to edit it - only to view
![prob2](https://cloud.githubusercontent.com/assets/19521910/17543491/4ccff3fa-5ed1-11e6-9ba8-9597f8b6b3f7.png)
![prob3](https://cloud.githubusercontent.com/assets/19521910/17543494/51912ea4-5ed1-11e6-9b8c-8d3dec760679.png)
Now I've lost the ability to do edit. I can only view record.
User - I logged in as an administrator with all privileges.
![admin1](https://cloud.githubusercontent.com/assets/19521910/17543500/5a233238-5ed1-11e6-85cc-db16e3fbdb36.png)
![admin2](https://cloud.githubusercontent.com/assets/19521910/17543502/5e3a8a6a-5ed1-11e6-8a54-09fe92fcabc2.png)

**Comment by serbiaserbia** [comment]
Illogical situation. see gif.

![animated2](https://cloud.githubusercontent.com/assets/19521910/17552273/3a7f2922-5eff-11e6-9c9d-4dca7d692201.gif)

**Comment by serbiaserbia** [comment]
I deleted workflow - now appeared a possibility to do edit.
I have to explore what is a problem

**Comment by nic86** [comment]
repeat the same steps you did in the demo version https://gitdeveloper.yetiforce.com/ if the problem persists then active the log files from config/debug.php in your server then checks for errors in log files in the cache/logs directory

**Comment by serbiaserbia** [comment]
I need to practice with workflow...  Its powerfull...  I made a mistake when creating workflow -my selection was Block edition 😀😀😀 that is why it didnt work 😅

Please can you make me clear for some functions in workflow:

1. Only once, when all conditions are met -this will run workflow -on create and modifing -but only once
2. System -I dont know when this will be executed?
3. Block edition - 😅ok I have practical experience...  Do you have an good example how to use this function?

**Comment by skramanzia** [comment]
+1 to create Wiki or forum to Workflow and configurations experience.

**Comment by paula-w** [comment]
You have to open a new issue with an appropriate title, since this is no longer a bug report, but a question instead. 
We believe in the "one question per issue"  rule to keep things in order.
